databases/pxlib/Portfile

```
@@ -28,22 +27,30 @@ checksums           rmd160  1bf8b7e14c92f47058aebd1a675b34a76b720660 \
                     sha256  4d0a84a36d46a81ff8c629c4a00de3e7b6e7327d5d56c0a2ae44c1ea715ae63f \
                     size    532895
 
-depends_build       port:pkgconfig \
-                    port:intltool \
-                    port:autoconf \
-                    port:automake \
-                    port:libtool
-
-depends_lib         port:libgsf \
-                    port:libiconv
-
 # reconfigure using autogen.sh for intltool 0.51 compatibility
 
 patchfiles          patch-autogen.sh.diff \
                     patch-configure.ac.diff
 
 configure.cmd       ./autogen.sh && ./configure
 
-configure.args      --disable-silent-rules \
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:gettext \
+                    port:intltool \
+                    port:libtool \
+                    port:pkgconfig
+
+depends_lib-append  port:libgsf \
+                    port:libiconv
+
+configure.args-append \
+                    --disable-silent-rules \
                     --enable-option-checking \
                     --with-gsf=${prefix}
+
+if {[string match *clang* ${configure.compiler}]} {
+    configure.cflags-append \
+                    -Wno-incompatible-function-pointer-types
```

Instead of hiding the error entirely, it might be better to just downgrade the error back to a warning:
```suggestion
                    -Wno-error=incompatible-function-pointer-types
```